src/App.tsx

```diff
@@ -1,34 +1,117 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { filteringTodo } from './services/filteringTodos';
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[] | null>(null);
+  const [query, setQuery] = useState('');
+  const [selectFiltering, setselectFiltering] = useState('');
```

Comment:
The state variable `selectFiltering` is initialized with an empty string. Consider initializing it with a default value like 'all', 'active', or 'completed' to align with the filtering options available in the `TodoFilter` component.